MabiCommerce/data/db/transports.json

```diff
@@ -31,5 +31,29 @@
     "Slots": 7,
     "Weight": 1700,
     "Id": 3
-  }
+  },
+  {
+    "Name": "Dog Sled",
+    "Icon": "data/img/transport/elephant.png",
```

Comment:
Can you add the actual images for these things? They should be in the pack files, or on the wiki

Reply:
I'll check the wiki, I've never messed with a pack file, lol.
If the wiki doesn't have them I'll look into it.

Reply:
Found icons on wiki, added to db/images/transports.